### PR TITLE
[Sprint Maintain] Add SwipeFresh List Leave Request

### PR DESCRIPTION
**Pull request by Tuanlvt**
Add SwipeFresh List Leave Request

### CODE REVIEW

File: app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestViewModel.java [hunk 1]
```diff
@@ -56,6 +57,7 @@
     @RequestType
     private int mRequestType;
     private QueryRequest mQueryRequest;
+    private boolean isLoading;
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestViewModel.java]: WARNING: [AvoidFieldNameMatchingMethodName] Field isLoading has the same name as a method
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName

File: app/src/main/res/layout/fragment_list_request.xml [hunk 2]
```diff
@@ -13,195 +13,201 @@
...
+                        android:layout_weight="0.7"
+                        android:gravity="center"
+                        android:orientation="horizontal"
+                        >
+                        <ImageButton
+                            android:layout_width="0dp"
+                            android:layout_height="@dimen/dp_40"
+                            android:layout_marginLeft="@dimen/dp_5"
+                            android:layout_marginStart="@dimen/dp_5"
+                            android:layout_weight="1"
```
Comment (trainer-android-fr) [inline app/src/main/res/layout/fragment_list_request.xml]: WARNING: [NestedWeights] Nested weights are bad for performance
 Layout weights require a widget to be measured twice. When a LinearLayout with non-zero weights is nested inside another LinearLayout with non-zero weights, then the number of measurements increase exponentially.

File: app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestViewModel.java (hunk 1 above)
Comment (nguyenhuy95dn) [inline app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestViewModel.java]: isLoading -> mIsLoading

File: app/src/main/res/layout/fragment_list_request.xml (hunk 2 above)
Comment (trainer-android-fr) [inline app/src/main/res/layout/fragment_list_request.xml]: WARNING: [NestedWeights] Nested weights are bad for performance
 Layout weights require a widget to be measured twice. When a LinearLayout with non-zero weights is nested inside another LinearLayout with non-zero weights, then the number of measurements increase exponentially.

**Pull request merged**